### ISSUE TITLE
ScreenSet Localization

### ISSUE THREAD

**Issue by benediktpascher** [SAP/gigya-react-native-plugin-for-sap-customer-data-cloud#14]
Is there a way to add params to the `showScreenSet` function as mentioned in the readme? I haven't found a way to retrieve the localized screenset.

**Comment by tal-mi** [comment]
Hi,
TS Interface is missing.
Will be fixed and available shortly in upcoming release.

Tal

**Comment by benediktpascher** [comment]
But looking at the function in `index.js`, `showScreenSet` is there twice so you're trying to make function overloading, which is (as far as I'm concerned) not possible with JavaScript.

**Comment by benediktpascher** [comment]
I guess you need to remove one of the implementations and replace it with something like:

```
/**
 * Show screen-set.
 *
 * @param {string} name
 * @param {map} params
 * @param {function} callback
 */
showScreenSet(name, params, callback) {
  const paramsIsFunc = typeof params === 'function';

  GigyaSdk.showScreenSet(name, !paramsIsFunc ? JSON.stringify(params) : "")

  listener = GigyaSdkEvents.addListener('event', (jsonData) => {
    const data = JSON.parse(jsonData)
    const callbackFunction = paramsIsFunc ? params : callback;
    callbackFunction(data.event, data.data)

    if(data.event == 'onCanceled' || data.event == 'onHide') {
      listener.remove()
    }
  })
}
```
to fully support both versions
- `function showScreenSet(name: string, callback: (event: string, data: GigyaDictionary) => void): void;`
- `function showScreenSet(name: string, params: Record<string, any>, callback: (event: string, data: GigyaDictionary) => void): void;`

**Comment by tal-mi** [comment]
Hi,

We will be taking a simpler approach in the upcoming version 0.0.7.
Will add an interface for opening the screenset with parameters.

**Comment by tal-mi** [comment]
Now available on version 0.0.7
